trainTestReinforcementAlgorithm.py

```diff
@@ -115,15 +138,39 @@ def saveRL(
     model.save(model_filename)  
 
 
+# %% [markdown]
+# ## Initialize Game Variables
+# %%
+# Set amount of time for training/testing. One step is one action for the snake.
+train_timesteps=1000
+test_timesteps=100
+
+# Set gym environment name
+env_name = 'snake-v0'
+
+# Set board dimensions
+board_height = 10
+board_width = 10
+
+# Set visualization arguments
+visualize_training = False # We don't want to visualize the training process
+visualize_testing = True # Set to true in order to see game moves in pygame. Should be false if run on server.
+visualization_fps = 30
+
+# Set reward function to be used in training 
+# Reward functions are defined in snakeRewardFuncs.py
+reward_function = basic_reward_func 
```

Comment:
where is `basic_reward_func` defined? Do we need to import reward_functions?

Reply:
We already import reward_functions, but maybe we need to do something like `reward_function = RewardFunctions.basic_reward_func`

Reply:
Yeah I like that idea Dave, I'll add it in for clarity!